### ISSUE TITLE
Question : If resizable = TRUE and colDef(width) are used, resizable disabled.

### ISSUE THREAD

**Issue by daeyoonlee** [glin/reactable#223]
Thanks for the great package.

Currently setting width in colDef seems to disable resizable. minwidth and maxwidth are also forced.
Is this intentional? I want to width only as an initial value and allow resizable. but there seems to be no way.
If only min width and maxwidth are used, resizable is possible, but the initial value is minwidth.

```
library(shiny)
library(reactable)

ui <- fluidPage(
  reactableOutput(outputId = "table"),
)

server <- function(input, output, session) {
  
  output$table <- renderReactable(
    reactable(
      data = iris,
      bordered = TRUE,
      resizable = TRUE,
      columns = list(
        Species = colDef(minWidth = 150, width = 200, maxWidth = 250)
      )
    )
  )
  
}

shinyApp(ui, server)
```
![image](https://user-images.githubusercontent.com/68131229/149702669-0f850187-ca89-41fc-b11b-fe940a3f52bf.png)

**Comment by glin** [comment]
Yes, this was intentional, or at least semi-intentional when resizing was reworked to respect the min and max widths of columns. Disabling resizing on fixed-width columns wasn't a primary intention, but came along with that rework. I can see how resizing would be useful on fixed-width columns though.

Unfortunately, resizing can't be easily reenabled on fixed-width columns without undoing the work to limit resizing to min/max widths. It's somewhat complicated, so I'll tag this as a feature request to look into later for now.

**Comment by daeyoonlee** [comment]
Thank you for the reply. If it is difficult to modify the `width` option, adding `initWidth` to the option seems to be an alternative.

**Comment by nteetor** [comment]
I strongly second having an initial or default width option. In my use case, I am loading saved column widths and either,
1) Users can no longer resize columns because I specify `width`
or
2) Users can no longer make columns smaller because I specify `minWidth`

**Comment by nteetor** [comment]
I've ~opened~ drafted a PR to hopefully address this issue.

**Comment by glin** [comment]
Done in #425:

> * `colDef()` gains an `initWidth` argument to set a column's initial width in pixels while keeping the column resizable. Unlike `width`, which sets a fixed width and disables resizing, `initWidth` only sets the starting width, so the column can still be resized within its `minWidth` and `maxWidth`. (@nteetor, [#223](https://github.com/glin/reactable/issues/223), [#374](https://github.com/glin/reactable/issues/374))
